src/Microsoft.Framework.PackageManager/Install/InstallGlobalCommand.cs

```diff
@@ -47,7 +47,7 @@ public FeedOptions FeedOptions
 
             if (packageIdAndVersion == null)
             {
-                WriteError("The name of the package to be installed was not specified.");
+                WriteError($"Installing {packageId} failed");
```

Comment:
`ResolvePackageIdAndVersion` returns `null` if the package id is null. That means that we cannot figure out from the result of that method if the package is null because the user didn't specify it or because it was not found. I suggest adding a check before we call that method. If the user passes a `null` package, give an appropriate error message and if the package is not found, give a different one